### DISCUSSION TITLE
Youtube videos not showing

### DISCUSSION THREAD

**Discussion by grunchoo** [requarks/wiki#4676] [Error / Bug Report]
when I edit the document I see the video but when I save and view it, it does not show it.

**Comment by ieaster1** [comment]
Have you enabled iframes for your pages?

In the Admin Area, Rendering -> Security -> Enable Allow iframes -> Apply

**Comment by Phil-hostwiki** [comment]
You can fix this with code injection. The solution is on this issue - -> https://github.com/requarks/wiki/discussions/4580#discussioncomment-1533379

**Reply by peteruithoven** [reply]
The issues are no longer accessible, can someone share the workaround here?

**Reply by Phil-hostwiki** [reply]
> The issues are no longer accessible, can someone share the workaround here?

Not sure why the issues were disabled.
Check if this works for you https://github.com/requarks/wiki/discussions/4580#discussioncomment-1533379
